internal/distro/fedora/package_sets.go

```diff
@@ -524,3 +525,19 @@ func containerPackageSet(t *imageType) rpmmd.PackageSet {
 
 	return ps
 }
+
+func bareMetalPackageSet(t *imageType) rpmmd.PackageSet {
+	return rpmmd.PackageSet{
+		Include: []string{
+			"policycoreutils",
+			"selinux-policy-targeted",
+			"kernel",
+			"firewalld",
+			"chrony",
+			"langpacks-en",
```

Comment:
I believe many of these packages are implicitly included, so this could be simplified.